eng/scripts/generate_aggregate_coverage_pom.py

```diff
@@ -29,6 +29,8 @@
 
 include_groups = []
 
+exclude_projects = ['com.azure.resourcemanager:azure-resourcemanager-samples']
```

Comment:
In Samples, jacoco is skipped in pom.xml.
https://github.com/Azure/azure-sdk-for-java/blob/21612b117a6f04f5cc51407c6d110c7db8df5a52/sdk/resourcemanager/azure-resourcemanager-samples/pom.xml#L39
Running jacoco:aggregate-report on it will fail.

Reply:
This actually works https://stackoverflow.com/a/68232557
However it had to be put to the generated POM.

The cause of problem might be that resourcemanager had some zip/war/jar in the samples.

So I think the solution is either exclude azure-resourcemanager-samples, or improve the generated POM to only include the classes we need.

@srnagar